frontend/dockerfile/docs/syntax.md

```diff
@@ -25,6 +25,39 @@ incrementing the major component of a version and you may want to pin the image
 change in between releases on labs channel, the old versions are guaranteed to be backward compatible.
 
 
+## Adding a git repository `ADD <git ref> <dir>`
+
+<!-- TODO: s/upstream-master/1.5/g after the release of 1.5 -->
+To use this instruction set Dockerfile version to `upstream-master`:
+
+```dockerfile
+# syntax=docker/dockerfile:upstream-master`
```

Comment:
It's `docker/dockerfile-upstream:master`

Reply:
fixed